### DISCUSSION TITLE
co-relate the objects

### DISCUSSION THREAD

**Discussion by calvinxu** [kubewharf/kelemetry#165] [Q&A] [answered]
As normal deployment within k8s, it is obvious that there is hierarchy among the deployment, replicaset, pod etc. thus Kelemetry may use the owerref to identify the event/audit log for the objects, however in certain use senario, when we deploy workloads for an application using helm charts, there might be no clear or obvious ownerefe among the object, however, they're all belong to the same application/service, so is it possible for Kelemetry today or in the future can support to manually tag or attach some labels on  these objects, then Kelemetry can utilize these tags/labels to process the span and eventually from k8s perspective, we can also have some kinds of relationship like what we have for deployment/replica/pods today?

**Comment by SOF3** [comment, marked as answer]
Yes, as mentioned in #164, you may write a specific linker plugin to support linking based on the helm chart annotations.
